nextflow_schema.json

```
@@ -97,6 +138,23 @@
                     "description": "Specify the distance at which the annotation script decides if a candidate is a circRNA or EI-circRNA.",
                     "help_text": "During annotation, if one of the start or end position of a circular candidate imperfectly overlaps an exon boundary, the script will consider positions within 'exon_boundary' (default 0bp) to be an exonic circRNA. If they fall outside of this range, the candidate is assumed to be an exonic-intronic circRNA, and the entire underlying sequence is taken for miRNA analysis, as opposed to just the exonic sequences for canonical exonic circRNAs. ",
                     "default": 0
+                },
+                "mirna_tools": {
+                    "type": "string",
+                    "fa_icon": "fas fa-wrench",
+                    "description": "Comma separated list of miRNA bindingsite prediction tools to use. Supported tools: miranda, targetscan.",
+                    "help_text": "Select one or a combination of miRNA bindingsite prediction tools for the pipeline e.g:\n--tool 'miranda, targetscan'",
+                    "default": "miranda, targetscan",
+                    "pattern": "^((miranda|targetscan|pita|tarpmir)?,?)*[^,]+$"
```

```suggestion
                    "pattern": "^((miranda|targetscan)?,?)*[^,]+$"
```
Not all of them are supported by now